runit-bionic-python/3.8/Dockerfile

```diff
@@ -4,7 +4,16 @@ LABEL maintainer="Socrata 'sysadmin@socrata.com'"
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get -y update && \
-    apt-get -y install python3.8 python3.8-dev python3-pip
+    apt-get -y purge python2.7 && apt-get -y autoremove --purge && \
```

Comment:
Out of curiosity, if we are removing Py2.7 would we also want to remove the existing Py3 (which I think is 3.6.something)?

Reply:
You would think that we would want to do that, but doing so breaks the existing pip install, even once it's pointed at Python 3.8